frontend/src/components/dashboard/Dashboard.js

```diff
@@ -182,9 +193,21 @@ const DropDownList = ({ buttonRef, handleOpen, handleClose, items, open }) => {
   );
 };
 
-export default function Dashboard({ className, headline, location, hubUrl }) {
+export default function Dashboard({ allHubs, hubData, className, headline, location, hubUrl }) {
   const classes = useStyles();
 
+  const [userOrganizations, setUserOrganizations] = useState(null);
+  const [isCreateIdeaOpen, setCreateIdeaOpen] = useState(false);
+  const token = new Cookies().get("token");
+
+  useEffect(async function () {
+    if (userOrganizations === null) {
+      setUserOrganizations("");
+      const userOrgsFromServer = await getUserOrganizations(token, locale);
+      setUserOrganizations(userOrgsFromServer || []);
+    }
+  });
```

Comment:
Does this have to run multiple times? Because if it only has to run once, I'd add the following suggestion:
```suggestion
  useEffect(async function () {
    if (userOrganizations === null) {
      setUserOrganizations("");
      const userOrgsFromServer = await getUserOrganizations(token, locale);
      setUserOrganizations(userOrgsFromServer || []);
    }
  }, []);
```

Reply:
True, I fixed it